### PR TITLE
QEMU v8: upgrade TF-A to v2.11

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

**Review by jforissier — COMMENTED** [review]
For commit "qemu_v8: build hafnium using OP-TEE's clang toolchain":

I suggest to replace "Use the toolchain provided in PATH instead" with "Instead, use OP-TEE's clang toolchain ('make clang-toolchains') if it exists, or the one already in PATH".

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: qemu_v8.mk [hunk 1]
```diff
@@ -161,9 +161,7 @@ TF_A_FLAGS ?= \
 	PLAT=qemu \
 	QEMU_USE_GIC_DRIVER=$(TFA_GIC_DRIVER) \
 	ENABLE_SVE_FOR_NS=2 \
-	ENABLE_SME_FOR_NS=2 \
 	ENABLE_SVE_FOR_SWD=1 \
-	ENABLE_SME_FOR_SWD=1 \
```
Comment (jforissier) [inline qemu_v8.mk:166]: Comment on "qemu_v8: enable SME for SPMC_AT_EL=2".
Why are these two lines not needed when SPMC_AT_EL=n? We still have QEMU_SME=on in that case.

Comment (jenswi-linaro) [reply qemu_v8.mk:166]: That's a good question. I'll restore the lines and remove the corresponding line below.

**Comment by jenswi-linaro** [comment]
Tags applied

**Review by jforissier — COMMENTED** [review]
For "qemu_v8: enable SME for SPMC_AT_EL=2":

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by jenswi-linaro** [comment]
Squashed and tag applied.

**Comment by jenswi-linaro** [comment]
Ready to merge?

**Pull request merged**